utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+kt_android_library(
+    name = "accessibility",
+    srcs = ["CustomAccessibilityManager.kt"],
+    visibility = ["//visibility:public"],
+)
+
+kt_android_library(
+    name = "impl",
+    srcs = [
+        "AccessibilityModule.kt",
+        "AndroidAccessibilityManager.kt",
+    ],
+    deps = [
+        ":accessibility",
+        artifact("com.google.dagger:dagger:2.27"),
```

Comment:
As discussed in chat, prefer using the third party wrappers instead of ``artifact`` since they offer a bunch of benefits (see #2663 for specifics).

Reply:
Done